e2e-tests/steps/reports.ts

```diff
@@ -0,0 +1,67 @@
+import { Page, expect } from '@playwright/test'
+import Excel, { CellValue } from 'exceljs'
+
+const reportTypeMetaData = {
+  submittedApplications: {
+    columnNames: [
+      'eventId',
+      'applicationId',
+      'personCrn',
+      'personNoms',
+      'referringPrisonCode',
+      'preferredAreas',
+      'hdcEligibilityDate',
+      'conditionalReleaseDate',
+      'submittedAt',
+      'submittedBy',
+      'startedAt',
+    ],
+    callToAction: "Download 'Submitted applications' report",
+  },
+  applicationStatusUpdates: {
+    columnNames: ['eventId', 'applicationId', 'personCrn', 'personNoms', 'updatedBy', 'updatedAt', 'newStatus'],
+    callToAction: "Download 'Application status updates' report",
+  },
+  unsubmittedApplications: {
+    columnNames: ['applicationId', 'personCrn', 'personNoms', 'startedBy', 'startedAt'],
+    callToAction: "Download 'Un-submitted applications' report",
```

Comment:
I guess this is coming from the API, but 'un-submitted' applications feels weird content wise.

Reply:
This matches the content in our FE, so we can change this at any point. I'll run it past Chanel 👍🏾

Reply:
Leaving as is for now, but the conversation is here: https://mojdt.slack.com/archives/C04T6SC31HA/p1709137791628899